specification/datashare/resource-manager/readme.go.md

```diff
@@ -32,5 +32,5 @@ These settings apply only when `--tag=package-2019-11-01 --go` is specified on t
 Please also specify `--go-sdk-folder=<path to the root directory of your azure-sdk-for-go clone>`.
 
 ``` yaml $(tag) == 'package-2019-11-01' && $(go)
-output-folder: $(go-sdk-folder)/services/datashare/mgmt/2019-11-01/$(namespace)
+output-folder: $(go-sdk-folder)/services/stable/datashare/mgmt/2019-11-01/$(namespace)
```

Comment:
Please revert this change. Go package path convention does not allow extra directory between `services` and the namespace of that service, unless for preview packages.